sr_port/ydberrors_ctl.c

```diff
@@ -29,7 +29,7 @@ LITDEF	err_msg ydberrors[] = {
 	{ "SUBSARRAYNULL", "Non-zero number of subscripts [!UL] specified but subscript array parameter is NULL in !AD call", 3 },
 	{ "MISSINGVARNAMES", "At least one variable name must be specified", 0 },
 	{ "TOOMANYVARNAMES", "Number of varnames specified exceeds maximum (!UL)", 1 },
-	{ "INVNAMECOUNT", "Invalid value for namecount parameter in a !AD call", 2 },
+	{ "INVNAMECOUNT", "namecount parameter in a !AD call cannot be a negative number", 2 },
```

Comment:
Maybe "The namecount parameter..." so sentence can begin with capital?

Reply:
This is already the case. Below is the latest error message and it begins with a capital letter.

     36 INVNAMECOUNT    <Number of varnames (namecount parameter in a !AD call) cannot be less than zero>/error/fao=2